app/views/schools/choose_programme/advisory.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :before_content, govuk_back_link(
+  text: "Back",
+  href: root_url) # what is the back path here, could we use an internal referer helper?
+%>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">Do you know how you want to run your statutory induction programme?</h1>
+      <p class="govuk-body">You will be choosing a programme for any new early career teachers. They will complete this programme over 2 years.</p>
+      <p class="govuk-body">Only use this service if your school wants to:</p>
+      <ul class="govuk-list govuk-list--bullet">
```

Comment:
`p`s and `ul` are not children of `h1`. Can we bring them 1 level of indentation to the left?